nx-X11/programs/Xserver/hw/nxagent/Extensions.c

```diff
@@ -284,9 +284,9 @@ static int nxagentRandRInitSizes(ScreenPtr pScreen)
                  1024,  768,  900,  900, 1200, 1050, 1080, 1200, 0, 0};
 */
   int w[] = {0, 320, 640, 640, 800, 800, 1024, 1024, 1152, 1280, 1280, 1280, 1360,
-                 1440, 1600, 1600, 1680, 1920, 1920, 0, 0};
+	     1440, 1600, 1600, 1680, 1920, 1920, 32767, 0, 0};
   int h[] = {0, 240, 360, 480, 480, 600,  600,  768,  864,  720,  800, 1024,  768,
-                  900,  900, 1200, 1050, 1080, 1200, 0, 0};
+	     900,  900, 1200, 1050, 1080, 1200, 32767, 0, 0};
```

Comment:
MAXSHORT ist better, agreed. We can add some more resolutions but with the new maximum size you can add any resolution on your own.

However: As long as rrxinerama is active your won't ever see those resolutions.